src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,39 +44,46 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        if (dateParams.length == 0) {
+            return Optional.empty();
+        }
```

Comment:
do we need this check considering we are catching an exception?

Reply:
removed